dev-tools/mage/common.go

```diff
@@ -216,12 +216,12 @@ func dockerInfo() (*DockerInfo, error) {
 	return &info, nil
 }
 
-// HaveDockerCompose returns an error if docker-compose is not found on the
+// HaveDockerCompose returns an error if docker is not found on the
 // PATH.
 func HaveDockerCompose() error {
-	_, err := exec.LookPath("docker-compose")
```

Comment:
Let's also make sure that compose plugin is installed by runnigg (`docker compose version`)

Reply:
> Let's also make sure that compose plugin is installed by runnigg (`docker compose version`)

What do you mean? Call `docker compose version`, parse the output and ensure the compose plugin is installed?

Reply:
I agree that this makes sense if the plugin is installed separately than docker itself. Otherwise, you are just testing that docker is available here, and not compose.